official/keras/resnet50_cifar10_model.py

```diff
@@ -0,0 +1,149 @@
+from __future__ import print_function
+
+import keras
+from keras import backend as K
+from keras.engine.topology import get_source_inputs
+from keras.layers import AveragePooling2D, Input, Flatten
+from keras.layers import Dense, Conv2D, BatchNormalization, Activation
+from keras.models import Model
```

Comment:
Prefer importing modules for user-facing code:

```python
import keras
from keras import layers
```

These two are basically all you need in this file.

Also: one import per line